doc/write_your_own/components/intro.rst

```diff
@@ -1,29 +1,71 @@
 Cube Browser Components
 =======================
 
+Construction
+------------
+
+**Decide on options and widgets**
+
+To start making your Explorer, you need to decide which aspects of cube plotting you would like to offer options for and which aspects you would like to hard code.
+The examples in the notebooks below show some of the options that you might want to offer, and the widgets that you could use to display them.
+The syntax to create each widget is very similar, but some small aspects (like the widget type) differ.  The basic outline looks like this:::
+
+    widget1 = ipywidgets.ToggleButtons(
+              description='Dimension:',
+              options=coordinates,
+              value = 'time')
+
+For more detailed guidance and a more complete list of widgets that are available, please visit the ipywidgets website.
+
+**Construct one widget per cell**
+
+Once you have decided what options and widgets you would like to present you can treat them as your individual components, each possessing a Jupyter notebook cell.
```

Comment:
I don't know, "possessing"? :ghost: 

Perhaps: "... each one contained in a Jupyter notebook cell".